activesupport/test/cache/stores/redis_cache_store_test.rb

```diff
@@ -274,14 +274,6 @@ def emulating_unavailability
   end
 
   class DeleteMatchedTest < StoreTest
-    test "deletes keys matching glob" do
-      @cache.write("foo", "bar")
-      @cache.write("fu", "baz")
-      @cache.delete_matched("foo*")
-      assert_not @cache.exist?("foo")
-      assert @cache.exist?("fu")
-    end
-
```

Comment:
Why delete this test?

Reply:
It's duplicated by the new test in local_cache_behavior.

Reply:
The new test is wrapped in `with_local_cache`, which changes the behaviour of `delete_matched`. I'd like to keep the separate test coverage for `delete_matched` without the local cache, since the local cache is not always active (e.g. in jobs, or on the console).